### ISSUE TITLE
Arena Deck Group

### ISSUE THREAD

**Issue by everchump** [Epix37/Hearthstone-Deck-Tracker#561]
i like to play arena and record the arena decks ,but now there is not a group just recording the arena decks.  the rank decks and the arena decks  edited in the same place ,will you change it ?

Sorry for my poor english, english is not my first language.

**Comment by Epix37** [comment]
I don't quite understand what you mean. Could you rephrase that?

**Comment by everchump** [comment]
Add a group just record arena decks

![Arena decks](http://i59.tinypic.com/23sewip.jpg)

**Comment by Epix37** [comment]
You can add "Arena"-tags to your arena decks (with the deck selected: *EDIT > QUICK SET TAG > ARENA*) and then filter your decks via *DECKS > QUICK FILTER > ARENA* to only show arena decks.

**Comment by everchump** [comment]
oh thank you
